kubesae/providers/aws.py

```diff
@@ -45,7 +46,73 @@ def configure_eks_kubeconfig(c, cluster=None, region=None):
         region = c.config.aws.get("region", "us-east-1")
     c.run(f"aws eks update-kubeconfig --name {cluster} --region {region}")
 
+@invoke.task(name="sync_media")
+def sync_media_tree(
+    c,
+    target_env="staging",
+    media_bucket="MEDIA_STORAGE_BUCKET_NAME",
+    acl="public-read",
+    local_target=None,
+    dry_run=False,
+    delete=False,
+):
+    """Sync an S3 media tree for a given environment/namespace to another. 
+
+    Args:
+        target_env   (string, required): A deployment host defined in ansible host_vars (e.g. "production", "staging", "dev"). DEFAULT: staging
```

Comment:
Not sure if it's just me, but from reading the name `sync_media_tree` I assumed that it would get all of the media from an environment and sync it to the machine that's running the command. If the task name sounds like that to anyone else, maybe it can be renamed, or maybe the default `target_env` should be local?
I'm not positive it needs to be changed, but want to avoid surprises for others.

Reply:
Maybe `target_bucket` instead of `target_env`? Or maybe a bit more explanation in the docstring about how this works?

Reply:
Sure, `target_bucket` can make it a little clearer, though the name `sync_media_tree` still doesn't sound sound like it's necessarily syncing between buckets. I think reading the docstring and args explains it well enough, though. My main confusion was reading the name of the task and incorrectly assuming what it does based on the name.

Reply:
Nice, I like the change